inst/assets/bibliography.bib

```diff
@@ -2290,4 +2290,21 @@ @article{Hang2021
     year = {2021},
     issn = {1474-760X},
     doi = {10.1186/s13059-021-02368-1},
-}
\ No newline at end of file
+}
+
+@Manual{Benedetti2024iseetree,
+    title = {{iSEEtree: Interactive visualisation for microbiome data}},
+    author = {Giulio Benedetti and Leo Lahti},
+    year = {2024},
+    note = {R package version 1.1.0},
+    url = {https://bioconductor.org/packages/iSEEtree},
+    doi = {10.18129/B9.bioc.iSEEtree},
+}
+
+@Manual{Benedetti2024miadash,
+    title = {{miaDash: Shiny app for interactive analysis and exploration of microbiome data}},
+    author = {Giulio Benedetti and Leo Lahti},
+    year = {2024},
+    note = {R package version 0.99.0},
+    url = {https://github.com/microbiome/miaDash},
+}
```

Comment:
Did you get support from others for the interactive app?

Reply:
Just Jeba. He's ctb in description